src/docs/asciidoc/reference_doc/card_notification.adoc

```diff
@@ -13,7 +13,14 @@ When a user receives a card, he is notified via a resume of the card on the left
 == Notification configuration
 
 For each process/state, the user can choose to be notified or not when receiving a card. If he chooses not to be
-notified then the card will not be visible in the feed. However, it will be visible in the archives screen.
+notified then the card will not be visible:
+
+* in the feed or the timeline
+* in the logging screen
```

Comment:
it will be visible in logging screen

Reply:
Thanks, I did my first test with the "task" usercard and it is not displayed in logging anyway, that's why. I'll correct it.